src/main/java/se/sundsvall/billingpreprocessor/service/MessagingService.java

```diff
@@ -17,15 +19,16 @@
 @Service
 public class MessagingService {
 	private static final boolean ASYNCHRONOUSLY = true;
+	private static final Logger LOG = LoggerFactory.getLogger(InvoiceFileService.class);
 
 	private final String applicationName;
 	private final String environment;
 	private final MessagingClient client;
 	private final ErrorMessageProperties properties;
 
 	public MessagingService(MessagingClient client,
-		@Value("${spring.application.name:}") String applicationName,
-		@Value("${spring.profiles.active:default}") String environment,
+		@Value("${spring.application.name:default}") String applicationName,
```

Comment:
Should the application start without an applicationName?

Reply:
The application will not start if `spring.application.name` is not present, so the `:` after name is unnecessary - will remove.